docs/reference/connectivity.md

```diff
@@ -73,6 +73,7 @@ The connection mode you are in will dictate the specific ports and URLs that nee
 | Auto | Auto-switches. See direct and relay modes. | Switch to direct or relay mode to troubleshoot. |
 | Direct | Hosts: A port in the range 5990 - 5999 needs to be opened to accept inbound local network connections.<br /><br />Guests: A network route and outbound access to the host on this same port. | Verify "vsls-agent" is not blocked by your desktop firewall software for this port range and that you can ping one another. While Windows and other desktop software should prompt you the first time the agent starts up, we have seen instances where group policies prevent this from happening and you will need to [manually add the entry](#manually-adding-a-firewall-entry). You may also be running into [proxy issues](#proxies) that need to be resolved. |
 | Relay | Outbound access to `*.servicebus.windows.net:443`. | Ensure your corporate or personal network firewall allows you to connect to this domain. You may also be running into [proxy issues](#proxies) that need to be resolved.|
+| Any | Outbound access to `*.online.visualstudio.com`. | Ensure your corporate or personal network firewall allows you to connect to this domain. Enter https://sts.online.visualstudio.com/api/swagger/index.html in a browser and verify you land at the swagger page. You may also be running into [proxy issues](#proxies) that need to be resolved.|
```

Comment:
Aren't we moving away from any use of VS Online?

Reply:
That's the question for Jason G but until it's changed we can recommend the customers to whitelist this URl.